portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,15 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+// function addBackground()  {
```

Comment:
nit: Generally I think we should avoid committing commented out code, unless we have  strong reason for leaving it in.

Reply:
Ok, I will take it out.